decentraland/unity-renderer
# feat: Chat Mentions

## What does this PR change?
Implementation of the new [Chat Mentions](https://www.notion.so/decentraland/PRD-Chat-Mentions-ffd23437a8c34cddadc401fabd7f5139) feature.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?explorer-branch=feat/chat-mentions
2. Open the chat window.
3. Open any of the chats (`nearby`, channle or DM) and start writting `@` in the text input.
4. You should view a list of users suggestions to select.
5. One you select one of them, the mention will be created in the text input.
6. Send the message and observe that the mention has been added into the message.
7. If you click on the mention (for both sides: yours and the other users that are reading your message), a the context menu will be open for that mentioned user.
8. You should also see the mentions in the notifications panel.
9. When someone send a mention to your user, you will see the corresponding entry (in the conversations list, in the chat window or in the notifications panel) with a different visual design that indicates that someone is mentioning you.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
